CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+on our [mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/virtual-biohackathon), other [related projects listed on our wiki page](https://github.com/virtual-biohackathons/covid-19-bh20/wiki) or via [slack conversations](https://virtualbiohac-xt62674.slack.com/join/shared_invite/zt-cuur40oj-wdrVz50NocwVrH7vgKTdPg).
+
+You can raise any issue relevant to this project/event publicly in the mailing list (google group), we will address and/or assess your change proposal as promptly as we can to make your participation easier.
+
+We have a [Code of Conduct](https://github.com/virtual-biohackathons/covid-19-bh20/blob/master/CODE_OF_CONDUCT.md) that applies to all the activities and projects involved in this event.
+For any organisation related queries or concern, you can directly reach out to the organisers Leyla Garcia (email: ljgarciaco@gmail.com), Pjotr Prins (email: pjotr.public708@thebird.nl) 
+and  Tazro Ohta (email: t.ohta@dbcls.rois.ac.jp). 
+
+- With such a large group of participants, we are encouraging a bottom-up approach, where groups are formed around projects/topics and work (kind of) independently from others. 
+- If you need any information from another group/topic, you can always reach people via email or slack (please note that as the hacking starts, Slack might become more interactive, you can also directly ping people there, be patient please and re-ping if needed or look for an alternative contact). 
+- Every group/topic will have leader/coordinator/facilitator, please check the wiki to get more information (as we get close to the hacking week, we expect the wiki to change quickly and people to add more group/topic details there). 
+- We also expect group/topic leaders/coordinators/facilitators to provide a quick (daily if possible) summary via the wiki so everybody can get an idea on it.
+
+## How can I participate at this event?
+
+Join the [Google group mailing list](https://groups.google.com/forum/#!forum/virtual-biohackathon) and the [virtualbiohackathoncovid19 slack](https://join.slack.com/t/virtualbiohac-xt62674/shared_invite/zt-cuur40oj-wdrVz50NocwVrH7vgKTdPg) and join the conversation! No approval needed. 
+
+## How can join the hackthon?
+- Please have a look at the [GitHub wiki](https://github.com/virtual-biohackathons/covid-19-bh20/wiki) to see the existing projects. 
+- When you’ve found a project you’d like to contribute to, 
+do a quick scan to make sure that the project is suitable for accepting contributions.
+- Commonly, channels are created on our Slack workspace, so please have a look there to see if the group/topic you are interested in is already there. 
+- If there is no channel, go to general or COVID-19 mailing list and raise a question on how to join that group/topic/project. Members working on that project would help you participate.
```

Comment:
Re: _general or COVID-19 mailing list_, isn't there just one, and isn't this the Google Group? Or are there others that I missed? Either way, this should probably be linked to what is the right answer?

Reply:
Instead of 
- If there is no channel, go to general or COVID-19 mailing list and raise a question on how to join that group/topic/project. Members working on that project would help you participate.
It should be
- If there is no channel, go to "general" slack channel or [COVID-19 Google group mailing list](https://groups.google.com/forum/#!forum/virtual-biohackathon) and raise a question on how to join that group/topic/project. Members working on that project would help you participate.

Reply:
@malvikasharan I tried to create a patch with this change (on your forked copy) but I got a GitHub error. Could you please add this change? Once that is done, I will accept and merge this PR. Thanks.